src/Libs/Soa/v1/Soa.php

```diff
@@ -0,0 +1,101 @@
+<?php declare(strict_types=1);
+
+namespace Coverzen\Components\YousignClient\Libs\Soa\v1;
+
+use Coverzen\Components\YousignClient\YousignClientServiceProvider;
+use Exception;
+use Illuminate\Http\Client\ConnectionException;
+use Illuminate\Http\Client\RequestException;
+use Illuminate\Http\Client\Response as ClientResponse;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Log;
+use RuntimeException;
+use function is_array;
```

Comment:
quale è l'utilita di avere use in_array? solo a livello "documetnale" per ch ilegge la classe? perche leggevo che con laravel adesso a livello prestazoini non abbiamo guadagni

Reply:
Sinceramente me lo fa phpstorm in automatico, e non mi dispiace la cosa. È vero che probabilmente non fa differenza, ma a me non dispiace vedere che funzioni usa la classe.

Reply:
si si concordo, infatti volevo sapere se era una cosa piu documentale o c''era altro dietro